apps/docs/content/components/SectionIntro/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Section intro
+description: Use section intro to provide a title, optional description and link to a new section in the page.
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image displaying a section intro with a title, description and a link.](https://github.com/primer/brand/assets/912236/719022fa-4694-42c0-8554-b1923c65739f)
+
+- Heading: The title of the section.
+- Description: A short description of the section.
+- Link: An optional link to expand on the content introduced.
```

Comment:
On this page's title description, it says an "optional description" but here it says an "optional link." Hence I assume both are optional. We should probably keep this consistent depending on the use case of this component.

Reply:
Good point, I'll rephrase the `Description` description 😆